pkg/web/apps/apps_service.go

```diff
@@ -26,9 +27,31 @@ func NewService(repository Repository) Service {
 func (a *appsService) GetApps() (*[]models.App, error) {
 	apps, err := a.repository.GetApps()
 
+	// Check for errors and return the appropriate error to the handler
 	if err != nil {
```

Comment:
All the error handling from line 30 to 41 can be simplified in my opinion by just returning the err from the repository layer and also checking if there are no apps in the repository layer and returning the `models.ErrNotFound` there

Reply:
@davidffrench done.